libsplinter/src/events/reactor.rs

```diff
@@ -29,11 +33,15 @@ use crate::events::{ReactorError, WebSocketError};
 pub struct Reactor {
     sender: Sender<ReactorMessage>,
     thread_handle: thread::JoinHandle<()>,
+    running: Arc<AtomicBool>,
```

Comment:
This change could be in a separate commit, with more explanation.

Reply:
updated